Symfony/CS/Tests/Fixer/Symfony/SilencedDeprecationErrorFixerTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Jules Pietri <jules@heahprod.com>
+ *
+ * @internal
+ */
+final class SilencedDeprecationErrorFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return array(
+            array(
+                '<?php trigger_error("This is a deprecation warning."); ?>',
+            ),
+            array(
+                '<?php trigger_error("This is a deprecation warning.", E_USER_WARNING); ?>',
+            ),
+            array(
+                '<?php @trigger_error("This is a deprecation warning.", E_USER_DEPRECATED); ?>',
```

Comment:
>and one to let as is

you don't need that one :)
let say you have the following test input for `makeTest`; array('A', 'B').
Then the test will first check 'B' is changed into 'A' by the fixer and than test if 'A' is not changed by fixer.
(for ref.: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.12/Symfony/CS/Tests/Fixer/AbstractFixerTestBase.php#L97)